sklearn/metrics/cluster/tests/test_unsupervised.py

```diff
@@ -185,24 +178,7 @@ def test_silhouette_nonzero_diag(dtype):
         silhouette_samples(dists, labels, metric='precomputed')
 
 
-def assert_raises_on_only_one_label(func):
-    """Assert message when there is only one label"""
-    rng = np.random.RandomState(seed=0)
-    with pytest.raises(ValueError, match="Number of labels is"):
-        func(rng.rand(10, 2), np.zeros(10))
-
-
-def assert_raises_on_all_points_same_cluster(func):
-    """Assert message when all point are in different clusters"""
-    rng = np.random.RandomState(seed=0)
-    with pytest.raises(ValueError, match="Number of labels is"):
-        func(rng.rand(10, 2), np.arange(10))
-
-
 def test_calinski_harabasz_score():
-    assert_raises_on_only_one_label(calinski_harabasz_score)
```

Comment:
I think we need an alternative test rather than no longer testing this case

Reply:
I'll write some tests